tutorial/TUTORIAL.md

```diff
@@ -841,20 +1049,27 @@ into effect.
 If you'd like to not return a `Box`, however, another option is to name the
 return type explicitly. For example:
 
-```rust
+```rust,no_run
+# extern crate futures;
+# use futures::{Future, Oneshot, oneshot, Poll};
 struct MyFuture {
     inner: Oneshot<i32>,
 }
 
 fn foo() -> MyFuture {
     let (tx, rx) = oneshot();
     // ...
-    MyFuture { inner: tx }
+    MyFuture { inner: rx }
```

Comment:
I think this was bug in the tutorial before. I guess `rustdoc` tests are awesome.